dependencies.gradle

```diff
@@ -3,9 +3,10 @@
 dependencies {
     implementation("curse.maven:cofh-lib-220333:2388748")
 
-    compileOnly 'com.github.GTNewHorizons:waila:1.7.3:dev'
-    compileOnly 'com.github.GTNewHorizons:NotEnoughItems:2.5.27-GTNH:dev'
-    compileOnly 'com.github.GTNewHorizons:OpenComputers:1.10.11-GTNH:dev'
+    compileOnly 'com.github.GTNewHorizons:waila:1.8.1:dev'
+    compileOnly 'com.github.GTNewHorizons:NotEnoughItems:2.6.14-GTNH:dev'
+    compileOnly 'com.github.GTNewHorizons:OpenComputers:1.10.16-GTNH:dev'
+    devOnlyNonPublishable("com.github.GTNewHorizons:GT5-Unofficial:5.09.45.168:dev")
```

Comment:
Why do you added dependency for GT? As I can see there's nothing *requiring* GregTech.

Reply:
Wanted to have it in dev env for testing IC2 energy integration